src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -2273,8 +2294,14 @@ protected override void BeginProcessing()
                     }
                 }
 
+                if (!string.IsNullOrEmpty(SecurityDescriptorSddl))
```

Comment:
would this work?

```csharp
service = new ServiceController(Name);

if (!string.IsNullOrEmpty(SecurityDescriptorSddl))
{
    SetServiceSecurityDescriptor(service, SecurityDescriptorSddl, hService);
}

Reply:
That's a parameter, right? Shouldn't we be using `if (MyInvocation.BoundParameters.ContainsKey(nameof(SecurityDescriptorSddl))) { ... }`

Reply:
@adityapatwardhan @vexx32  yes, I've used the same way how its implemented for `Set-Service`. I referred

L1883 `if (!string.IsNullOrEmpty(Status))` which is used for `-Status` parameter in `Set-Service`.